profile.graphqls

```diff
@@ -84,6 +84,28 @@ type ProfileTask {
     logs: [ProfileTaskLog!]!
 }
 
+# Profile thread stack anayze tree
+type ThreadStackAnalyzation {
+    # stack code signature
+    codeSignature: String!
+    # self include children duration(millisecond)
+    duration: Int!
+    # self exclude children duration(millisecond)
+    durationChildExcluded: Int!
+    # total dump count
+    count: Int!
+    # max recursion invoke count
+    maxRecursionCount: Int!
```

Comment:
Don't recommend use this. Stack depth is limited in the agent. We could say it is safe here.